account_invoice_crm_tag/report/account_invoice_report.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Tecnativa - Stefan Ungureanu
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountInvoiceReport(models.Model):
+    _inherit = "account.invoice.report"
+
+    tag_ids = fields.Many2many(
+        comodel_name="crm.tag",
+        relation="account_move_crm_tag_rel",
+        column1="account_move_id",
+        column2="crm_tag_id",
+        string="Invoice CRM Tags",
+        readonly=True,
+    )
+
+    def _select(self):
+        select_str = super()._select()
+        return "%s, account_move_crm_tag_rel.crm_tag_id as tag_ids" % select_str
```

Comment:
Seeing https://github.com/odoo/odoo/blob/15.0/addons/project/report/project_report.py, there's no need of adding it to the select.

Reply:
Changes done, I updated the from insertion more accordingly with your reference.

Reply:
Isn't just declaring the orm m2m field enough?

Reply:
@chienandalu You are right, thank you